src/components/SearchBar/SearchBar.module.css

```diff
@@ -0,0 +1,11 @@
+/*------------------------------------*\
+    # SEARCH FIELD
+\*------------------------------------*/
+
+/**
+ * Search bar wrapper for the SearchField and SearchButton
+ */
+.search-bar {
+  display: flex;
+  gap: var(--eds-size-1);
```

Comment:
How does this look on mobile? Do you think we should consider wrapping the button to the next line on narrower windows?

Reply:
Oh good question, I'll ask design how they want it to look

Reply:
How it looks on mobile: ![image](https://user-images.githubusercontent.com/86632227/187788052-bcf6d1dd-0cfb-441f-8c65-a987532443a5.png)

And confirmed with design that this is how they want it

Reply:
Awesome, thanks!